src/encoded/tests/test_audit_file.py

```diff
@@ -249,6 +249,39 @@ def test_audit_file_mismatched_paired_with(testapp, file1, file4):
                'inconsistent paired_with' for error in errors_list)
 
 
+def test_audit_paired_with_non_fastq(testapp, file1, file6):
+    testapp.patch_json(file1['@id'], {
+                        'run_type': 'paired-ended',
+                        'paired_end': '1'})
+    testapp.patch_json(file6['@id'], {
+                        'run_type': 'paired-ended',
+                        'paired_end': '2',
+                        'paired_with': file1['uuid']})
```

Comment:
Missing `platform` here. It is required by the `run_type` property: https://github.com/ENCODE-DCC/encoded/blob/dev/src/encoded/schemas/file.json#L148-L150

You can patch it to platform1 like file4 above.

Reply:
DONE